compiler/src/iree/compiler/Dialect/Flow/Transforms/RaiseSpecialOps.cpp

```diff
@@ -27,13 +28,64 @@ namespace Flow {
 
 namespace {
 
+// Method to match a transpose operation.
+static bool match2DTranspose(linalg::LinalgOp genericOp) {
+  if (genericOp.getNumDpsInputs() != 1 || genericOp.getNumDpsInits() != 1) {
+    return false;
+  }
+  // Check only for 2D ops.
+  if (genericOp.getNumLoops() != 2 ||
+      genericOp.getNumLoops() != genericOp.getNumParallelLoops()) {
+    return false;
+  }
+  // Check for transpose map.
+  AffineExpr d0, d1;
+  MLIRContext *context = genericOp.getContext();
+  bindDims(context, d0, d1);
+  SmallVector<AffineMap> expectedMaps = {
+      AffineMap::get(2, 0, {d0, d1}, context),
+      AffineMap::get(2, 0, {d1, d0}, context)};
+  if (genericOp.getIndexingMapsArray() != expectedMaps) {
+    return false;
+  }
+
+  // Check the body.
```

Comment:
nit: Remove the commiet? This comment is redundant and not explains anything. The below code describes `check the body` itself.